src/drivers/wayland/lv_wayland.h

```diff
@@ -0,0 +1,153 @@
+/*******************************************************************
+ *
+ * lv_wayland.h - Public functions of the LVGL Wayland client
+ *
+ * Based on the original file from the repository.
+ *
+ * Porting to LVGL 9.1
+ * Copyright (c) 2024 EDGEMTech Ltd.
+ *
+ * See LICENSE.txt for details
```

Comment:
I don't think LVGL has a LICENCE.txt. Should there be a link to a EDGEMTech repo?

Reply:
I'm not entirely sure what it means to have a copyright in an MIT licensed repo, where the copyright refers to a company other than the actual owner. What are the practical implications of having this copyright?

Reply:
@kisvegabor EDGEMTech directives, if a change is made to a file the header must explain the change(s) and then mention the author of the changes, company + email. This change is Copyright 2024 but under the MIT license. This common practice in the Linux kernel or Zephyr. https://github.com/torvalds/linux/blob/master/kernel/irq/manage.c

Reply:
Hm, okay then.